js/details.js

```diff
@@ -64,33 +65,11 @@
           }, this);
         }, this);
 
-        var actions = [
-          {
-            title: 'Install this theme',
-            action: 'install'
-          },
-          {
-            title: 'Fork this theme',
-            action: 'fork'
-          },
-          {
-            title: 'Remove this theme',
-            action: 'remove',
-            css: 'destructive'
-          }
-        ];
-        actions.forEach(function(params) {
-          var link = document.createElement('a');
-          link.classList.add('action');
-          link.dataset.action = params.action;
-          if (params.css) {
-            link.classList.add(params.css);
-          }
-          var title = document.createElement('h3');
-          title.textContent = params.title;
-          link.appendChild(title);
-          this.list.appendChild(link);
-        }, this);
+        if (!currentTheme.manifestURL) {
+          this.enableButton.classList.add('disabled');
+        } else {
+          this.enableButton.classList.remove('disabled');
+        }
```

Comment:
```.disabled``` is set at https://github.com/hfiguiere/studio/blob/master/js/details.js#L128 in the action handler.

I'll deal with the rest.